client/landing/gutenboarding/onboarding-block/stepper-wizard.tsx

```diff
@@ -18,23 +19,30 @@ interface Props {
 
 const StepperWizard: FunctionComponent< Props > = ( { stepComponents } ) => {
 	const [ activeStep, setActiveStep ] = useState( 0 );
-
 	const handleNext = () => setActiveStep( activeStep + 1 );
 
+	const visibleStepCount = stepComponents.filter( step => !! step ).length;
+	const translatePercent =
+		( 100 / visibleStepCount ) * Math.min( activeStep, visibleStepCount - 1 ); // sometimes activeStep is invisible because of current Next handling
+	const springProps = useSpring( {
+		transform: `translate(0, -${ translatePercent }%)`,
+	} );
```

Comment:
This is hard (for me) to understand, and the comment doesn't help much 😅 
What's going on here?